packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenters/upgrade-range/translations/Messages_en_US.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Also I'm not sure we're supposed to have `en_US` file anymore

Reply:
I forgot to remove them on previous(separate) tasks.

```sh
$ find . -name "Messages_en_US.json" | wc -l
14
```

Reply:
Addressed here #5469.